tests/runner.py

```diff
@@ -1354,30 +1354,39 @@ class BrowserCore(RunnerCore):
   def __init__(self, *args, **kwargs):
     super(BrowserCore, self).__init__(*args, **kwargs)
 
+  def browser_open(url):
```

Comment:
Doesn't this need `@classmethod` too?

Reply:
No, since it doesn't call any other instance methods, but happy to add it if you'd prefer to standardize.

Reply:
Unless you use `@classmethod` then the first arg needs to be `self` no?

Reply:
No, it doesn't need it since it doesn't call any other instance methods.

Reply:
But its how this is invoked.  Normal methods take an instance as argument 0 right?

I think you need to make this a `@staticmethod` otherwise is doesn't make sense to the reader does it?

I've certainly never seen a static method in python without any decorator like this.  Have you?

Reply:
Ah, I see. I must admit I haven't used Python very much in my career, I didn't understand the nuances between normal instance methods, `@staticmethod`s and `@classmethod`s. I've read up on it now and gone with the `@staticmethod` decorator to fix this.